### ISSUE TITLE
New release compatible with OmniAuth 2.x?

### ISSUE THREAD

**Issue by ahukkanen** [omniauth/omniauth_openid_connect#106]
Could you cut a new release and push it to RubyGems?

We can see from the commit history that the gem has already been updated for OmniAuth 2.x compatibility but the `0.3.5` version at RubyGems is not compatible with OmniAuth 2.x.

![Version 0.3.5 OmniAuth compatibility](https://user-images.githubusercontent.com/864340/152568387-a7a5fd27-ad94-4508-9304-9383d3946092.png)

https://rubygems.org/gems/omniauth_openid_connect/versions/0.3.5

**Comment by Eric-Guo** [comment]
You can using gitlab-omniauth-openid-connect as temporary solution.

```diff
-gem 'omniauth_openid_connect'
+gem 'gitlab-omniauth-openid-connect', '~> 0.9.1', require: 'omniauth_openid_connect'
```

I think @stanhu will release the new version of `omniauth_openid_connect` when ready. (he is author of gitlab-omniauth-openid-connect)

**Comment by stanhu** [comment]
Released v0.4.0.
